lang/ruby/patches/001-fix-build-with-LibreSSL-over-v3.5.patch

```diff
@@ -0,0 +1,11 @@
+--- a/ext/openssl/ossl_pkey.c
```

Comment:
This patch comes from upstream? 
- If not, are you going to upstream this? 
- If yes, who is the author? Where is the commit message?

Reply:
I believe the upstream fix is this. And it is a little bit different. https://github.com/ruby/openssl/commit/e25fb0d0d86da5a9398ebdc9216b2ea89f80fa3d

Reply:
I just found the patch from openbsd source. Gladly to see that upstream was fixed it. May i close this pr and wait for sync from upsteam? @luizluca

Reply:
If you are facing the issue, you can just update this PR with the upstream patch. It might take some time until a new ruby is released with the fix incorporated and a little bit more to have the ruby package updated. I don't see any problem backporting fixes until then. The problem was only the use of a patch without any reference, comment, source, author.

Reply:
Okay, Thanks. I will refresh the patch with upstream commit.
Done it.